README.md

```diff
@@ -155,11 +155,17 @@ cmake .. -DETHASHCUDA=ON -DETHASHCL=OFF
 - `-DETHDBUS=ON` - enable D-Bus support, `OFF` by default.
 
 
-## Maintainer
+## Maintainers & Authors
 
 [![Gitter](https://img.shields.io/gitter/room/ethereum-mining/ethminer.svg)][Gitter]
 
-- Paweł Bylica [@chfast](https://github.com/chfast)
+[Contributors statistics since 2015-08-20](https://github.com/ethereum-mining/ethminer/graphs/contributors?from=2015-08-20).
+
+| Andrea Lanfranchi | [@AndreaLanfranchi](https://github.com/AndreaLanfranchi) |  |
+| Genoil            | [@Genoil](https://github.com/Genoil)                     |  |
+| Jean Cyr     | [@jean-m-cyr](https://github.com/jean-m-cyr) |  |
+| Paweł Bylica | [@chfast](https://github.com/chfast)         | 0x8FB24C5b5a75887b429d886DBb57fd053D4CF3a2 |
```

Comment:
Nice idea to publish a tip address. Nevertheless I think should be defined it's an Ether address. All ethash addresses look pretty much the same and there is the risk to send, say, PIRL to an ethaddress so it will never get received. :)

Reply:
Added "ETH: " prefix.